lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -961,34 +963,38 @@ sub ansible_show_status {
     my $sapcontrol_cmd = "$sapcontrol_env $sapcontrol_path/sapcontrol";
     my $instance_id = get_sdaf_instance_id(pattern => 'SCS');
     my $function = '';
-    if (grep(/ensa/, @{$args{scenarios}})) {
-        # Get instance ID
-        $instance_id = get_sdaf_instance_id(pattern => 'SCS');
-        $host_group = "$args{sap_sid}_SCS";
-
-        push @reports, {title => 'ENSA2 cluster', text => ansible_execute_command(command => 'sudo crm status full', host_group => $host_group, %common_args)};
-        $function = 'HACheckConfig';
-        push @reports, {title => "ENSA2 $function", text => ansible_execute_command(command => "$sapcontrol_cmd -nr $instance_id -function $function", host_group => $host_group, %common_args)};
-        $function = 'HACheckFailoverConfig';
-        push @reports, {title => "ENSA2 $function", text => ansible_execute_command(command => "$sapcontrol_cmd -nr $instance_id -function $function", host_group => $host_group, %common_args)};
-    }
 
-    # Show NW processes for each type of instance
-    if (grep(/nw/, @{$args{scenarios}})) {
-        foreach my $pattern (qw(PAS ERS SCS)) {
-            my $pattern_lc = lc($pattern);
-            # Get instance ID
-            $instance_id = get_sdaf_instance_id(pattern => $pattern);
-            $host_group = "$args{sap_sid}_$pattern";
-
-            push @reports, {title => "NW $pattern processes", text => ansible_execute_command(command => 'ps -ef | grep sap', host_group => $host_group, %common_args)};
-            # Add 'proceed_on_failure => 1' for 'GetProcessList' as it returns 'rc=3' (RC 3 = all processes GREEN)
-            $function = 'GetProcessList';
-            push @reports, {title => "NW $pattern $function", text => ansible_execute_command(command => "$sapcontrol_cmd -nr $instance_id -function $function", host_group => $host_group, %common_args, proceed_on_failure => 1)};
-            $function = 'GetSystemInstanceList';
-            push @reports, {title => "NW $pattern $function", text => ansible_execute_command(command => "$sapcontrol_cmd -nr $instance_id -function $function", host_group => $host_group, %common_args)};
-        }
-    }
+    # All ENSA2 checks will fail becuase sites are flipped after deployment
+    record_soft_failure('gh#sdaf-suse/sap-automation#9');
+    # if (grep(/ensa/, @{$args{scenarios}})) {
```

Comment:
Is there any other approach than commenting the code? Is it possible to detect that the bug is present or not?